### DISCUSSION TITLE
Let sympy use log2(x) instead of log(x)/log(2)

### DISCUSSION THREAD

**Discussion by nerai** [MilesCranmer/PySR#710] [Ideas]
The export to sympy currently translates log2(x) to log(x)/log(2). This is cumbersome if log2 is commonly used and makes the equations hard to read.

Sympy supports log2 natively, so I feel it would be appropriate to also translate log2 into that instead. It is perhaps possible to only fix the output in sympy, but ideally, it should be done in PySR directly.

**Comment by MilesCranmer** [comment]
Moved to https://github.com/MilesCranmer/PySR/issues/711
